server.py

```diff
@@ -4,7 +4,7 @@
 from flask import Flask, send_from_directory, abort, jsonify, request
 
 BUILD_FOLD = os.environ.get('BUILD_FOLD', 'frontend/build/')
-IMAGES_FOLD = os.environ.get('IMAGES_FOLD', '/images')
+IMAGES_FOLD = os.environ.get('IMAGES_FOLD', 'images')
```

Comment:
It won't be `images`. We will store the raw images at the absolute path `/images`, via a Docker volume mount.